examples/client/client.c

```diff
@@ -3045,8 +3045,11 @@ THREAD_RETURN WOLFSSL_THREAD client_test(void* args)
 
     if (dtlsUDP == 0) {           /* don't send alert after "break" command */
         ret = wolfSSL_shutdown(ssl);
-        if (wc_shutdown && ret == WOLFSSL_SHUTDOWN_NOT_DONE)
-            wolfSSL_shutdown(ssl);    /* bidirectional shutdown */
+        while (wc_shutdown && ret == WOLFSSL_SHUTDOWN_NOT_DONE) {
```

Comment:
polling does not look right.  Why would we expect a full shutdown in all cases?

Reply:
Only if `wc_shutdown` is set, which is set by the example option
`-w          Wait for bidirectional shutdown`